### PR TITLE
feat: 🚀 added glass panel color tinting

### PR DESCRIPTION
**Pull request by janbadura**
Added colour tinting of sidebar glass-panel

### CODE REVIEW

**Comment by janbadura** [comment]
Added "fix: firefox glass support" to this branch, as it is in the same ticket.

File: src/App.tsx [hunk 1]
```diff
@@ -596,11 +593,20 @@ const Sidebar = (theme: Theme) => `
...
-    backdrop-filter: blur(40px);
-    -webkit-backdrop-filter: blur(40px);
+    background: rgb(255, 255, 255);
+  }
+
+  /* if backdrop support: very transparent and blurred */
+  @supports ((-webkit-backdrop-filter: blur(40px)) or (backdrop-filter: blur(40px))) {
+    .feedbot-wrapper {
+      max-height: 100%;
+      background: linear-gradient(45deg, ${theme.mainColor}33, rgba(256,256,256, 0.8));
```
Comment (dvorakjan) [inline src/App.tsx:606]: What is the `33` constant here? And does it support `theme.mainColor` to be in both common color formats `rgb` and `hex`?

File: src/App.tsx [hunk 2]
```diff
@@ -564,9 +564,6 @@ const Sidebar = (theme: Theme) => `
     background-size: 15px;
     background-position: center center;    
 
-    /*backdrop-filter: blur(40px);
-    -webkit-backdrop-filter: blur(40px);
-    background-color: rgba(256,256,256, 0.5) !important;*/
```
Comment (dvorakjan) [inline src/App.tsx:569]: Can get rid of this in favor of new `@supports` section it is left here intentionally?

**Review by dvorakjan — CHANGES_REQUESTED** [review]

File: src/App.tsx (hunk 1 above)
Comment (janbadura) [reply src/App.tsx:606]: The 33 after ```theme.mainColor``` is the opacity constant in HEX, it's used here so we don't have to covert hex to rgba. I can comment on it above to explain it, or come up with some converting logic.

File: src/App.tsx (hunk 2 above)
Comment (janbadura) [reply src/App.tsx:569]: I've deleted this comment in some other PR I think :D but I can delete it here too.

File: src/App.tsx (hunk 1 above)
Comment (dvorakjan) [reply src/App.tsx]: Oh, my silly mistake. Anyway `rgb` is used format so please wrap `theme.mainColor` by some function which will append `33` when string starting with `#` or append float opacity as the last argument when color in `rgb(x,y,z)` format.

**Review by dvorakjan — DISMISSED** [review]

**Pull request merged**